lua/wikis/commons/OpponentDisplay.lua

```diff
@@ -194,13 +195,14 @@ end
 ---@field note string|number|nil
 ---@field showFaction boolean?
 ---@field showTbd boolean?
+---@field additionalClasses string[]?
 
 --[[
 Displays an opponent as a block element. The width of the component is
 determined by its layout context, and not of the opponent.
 ]]
 ---@param props BlockOpponentProps
----@return Html
+---@return Widget
```

Comment:
we have to check all consumers (on git and outside of git) to make sure this doesn't break anything
(e.g. :addClass / :css /... applied to it)